scripts/Kconfig

```diff
@@ -38,6 +38,12 @@ choice
                 select CAPI10
                 help
                   Semptian NSA121B has ethernet and 8GB DDR4 SDRAM. Uses Xilinx FPGA XCKU115.
+
+        config 8K5
+                bool "AlphaData ADM-PCIE-8K5, 16GB DDR4 SDRAM and Xilinx KU115 FPGA"
```

Comment:
Since the support for building an image for 8K5 is missing (like PSL DCP checking in [snap_check_psl](https://github.com/open-power/snap/blob/d2970258cf005abaf135e4480393d851a9d88da1/hardware/snap_check_psl)) I'd recommend to mark this card as experimental (like we did it for N250SP).